core/swift3Action/spm-build/_Whisk.swift

```diff
@@ -24,27 +24,40 @@ class Whisk {
         let strBlocking = blocking ? "true" : "false"
         let path = "/api/v1/namespaces/\(parsedAction.namespace)/actions/\(parsedAction.name)?blocking=\(strBlocking)"
 
-        return postSyncronish(uriPath: path, params: params)
+        return sendWhiskRequestSyncronish(uriPath: path, params: params, method: "post")
     }
 
     class func trigger(eventNamed event : String, withParameters params : [String:Any]) -> [String:Any] {
         let parsedEvent = parseQualifiedName(name: event)
         let path = "/api/v1/namespaces/\(parsedEvent.namespace)/triggers/\(parsedEvent.name)?blocking=true"
 
-        return postSyncronish(uriPath: path, params: params)
+        return sendWhiskRequestSyncronish(uriPath: path, params: params, method: "post")
+    }
+
+    class func createTrigger(triggerNamed trigger: String, withParameters params : [String:Any]) -> [String:Any] {
```

Comment:
Yep, I struggled to get that one to work - it kept bailing when I tried to pass down the feed annotation, so I parked it.

Didn't like `annotations: ["key":"feed","value":"/_/alarms/alarm"]`

Would throw a Malformed parameter exception from somewhere downstream ...